### DISCUSSION TITLE
Why are Classic and Lazer still different leaderboards?

### DISCUSSION THREAD

**Discussion by Silicove** [ppy/osu#23619] [Q&A]
I've started using Osu Lazer over Classic a while ago because I'm decaying and don't really care about PP, but one thing that bugs me about Lazer is the leaderboards and score submission on them as a whole. If you look at a map like Big Black in-game on Lazer, you'll see that the leaderboard is full of scores that definitely don't match up with the Stable version, which confuses me because supposedly you can now get PP on Lazer. On top of this, some of the top scores literally are just one note with somehow a 9 star rating that someone 4 modded, like how has this not been prioritized over any other change? This seems like a pretty big flaw that could definitely deter some people away

![osu_2023-05-21_13-50-08](https://github.com/ppy/osu/assets/94876381/391d22de-522d-4416-97cb-68e8b6b3d3f6)

**Comment by bdach** [comment]
> If you look at a map like Big Black in-game on Lazer, you'll see that the leaderboard is full of scores that definitely don't match up with the Stable version, which confuses me because supposedly you can now get PP on Lazer. 

Please watch https://www.youtube.com/watch?v=LgXrhrFFClY for the explanation. Or read https://osu.ppy.sh/wiki/en/Help_centre/Upgrading_to_lazer which is literally linked *in lazer settings* as a huge orange button. I can also act indignant, how much more do we need to explain and preempt and _prominently_ display things so that they are read/watched/understood?

> On top of this, some of the top scores literally are just one note with somehow a 9 star rating that someone 4 modded, like how has this not been prioritized over any other change?

lazer scores are going to be subject to wipes anyway. You can report such scores as fraudulent if you care enough but this problem will be addressed promptly.

**Comment by monochrome22** [comment]
Work is still being done on figuring out exactly how leaderboards on lazer should work. Lazer has new mechanics which make it difficult to compare lazer scores to stable scores. This was discussed in a community meeting which you can read about [here](https://docs.google.com/document/d/13cMCrQN4vvaQFA59BmFHk6D7PH9fwT2ANmhkm5cmzxE/edit).
So, because things are still being figured out, the leaderboards can look less than ideal for now. Leaderboard wipes will start happening once a path forward is determined and changes are implemented. 
That's why removing broken scores is not being prioritized; because even if they were removed, the leaderboards would still be flawed, and the broken scores will get removed with wipes anyway.
